packages/server/package.json

```diff
@@ -5,21 +5,21 @@
   "main": "index.ts",
   "type": "module",
   "scripts": {
-    "dev": "ts-node index.ts",
+    "dev": "wait-port 8545 && tsx index.ts",
```

Comment:
`tsx` works much better than `ts-node` in my experience; and we need to make sure the local node is running before starting the node server